tests/BitcoinCash/TWBitcoinCashTests.cpp

```diff
@@ -132,9 +132,9 @@ TEST(BitcoinCash, SignTransaction) {
     ASSERT_EQ(hex(output.encoded()),
         "01000000"
         "01"
-            "e28c2b955293159898e34c6840d99bf4d390e2ee1c6f606939f18ee1e2000d05" "02000000" "6b483045022100b70d158b43cbcded60e6977e93f9a84966bc0cec6f2dfd1463d1223a90563f0d02207548d081069de570a494d0967ba388ff02641d91cadb060587ead95a98d4e3534121038eab72ec78e639d02758e7860cdec018b49498c307791f785aa3019622f4ea5b" "ffffffff"
+            "e28c2b955293159898e34c6840d99bf4d390e2ee1c6f606939f18ee1e2000d05" "02000000" "6b483045022100ec09291f956567ac3fbf632710a44e4bdf5805d2167869e7d84cd95032e1af9c022019e912584e78dd62eeb5a72d0fabce18c8b4d6a475b8319ac13aaa429c0af8944121038eab72ec78e639d02758e7860cdec018b49498c307791f785aa3019622f4ea5b" "ffffffff"
         "02"
             "5802000000000000" "1976a914769bdff96a02f9135a1d19b749db6a78fe07dc9088ac"
-            "e510000000000000" "1976a9149e089b6889e032d46e3b915a3392edfd616fb1c488ac"
+            "e410000000000000" "1976a9149e089b6889e032d46e3b915a3392edfd616fb1c488ac"
```

Comment:
bitcoin cash doesn't have segwit at all, why this test is changed?

Reply:
Indeed, BCH has no segwit, I faintly remember the reason for BCH fork was dispute about block size...
I will check, but I think fee got a bit more precise, maybe a couple of bytes, but no segwit.

Reply:
Fee estimation adjusted for non-segwit case, so the 1 byte diff is gone (227 vs 226), BitcoinCash bhehaviour is the same, BCH test has been reverted.